components/TextFields/styling/README.md

```diff
@@ -0,0 +1,77 @@
+#  Text field styles
+
+MDC's text fields are designed to be styled by controllers implementing the `MDCTextInputController` protocol. The following controllers are included in the text field component and each one is highly customizeable to express your app's branding and styling.
+
+Some of the classes are marked as "Preferred". These are known, thru extensive UX research, to have big performance gains in usability.
+
+Another two of the classes are "Legacy". They are included to support older designs and avoid breaking changes that come from the size of newer styles.
+
+## MDCTextInputControllerOutlined (Preferred)
+
+![MDCTextInputControllerOutlined](docs/assets/MDCTextInputControllerOutlined.gif)
+
+- Stroked border
+- Border-crossing, floating placeholder
+- No fill
+
+### Notes:
+This class is intended for single line text fields only. For a visually compatible multi-line controller, use `MDCTextInputControllerOutlinedTextArea`.
+
+## MDCTextInputControllerFilled (Preferred)
+
+![MDCTextInputControllerFilled](docs/assets/MDCTextInputControllerFilled.gif)
+
+- Optionally stroked border / underline
```

Comment:
Gramma nit: "Optional stroked border / underline" makes it clearer that "optionally" is not an adverb for "floating". Maybe something like "Stroked border / underline (*optional*)" as a standard way to mark options?

Reply:
I don't think that 'Optional' could be confused for having something to do with floating. They are on separate lines with separate bullets.

Reply:
Yes, sorry. I probably mixed-up which optionally I was talking about. I swung back to this one after commenting on the "optionally floating" below.  Here I was trying to suggest a parenthetical for optional styles.